fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/api/IDatabaseAdapter.java

```diff
@@ -185,7 +185,7 @@ public void createIndex(String schemaName, String tableName, String indexName, S
      * @param tableName
      * @param predicate
      */
-    public void createPermission(String schemaName, String permissionName, String tableName, String predicate);
+    public void createOrReplacePermission(String schemaName, String permissionName, String tableName, String predicate);
```

Comment:
Is the purpose of this change to make it possible to run grant access multiple times for the same tenant?

Reply:
no, its so that we can call it after updating the table (e.g. adding or removing columns).
i found that without calling `createOrReplacePermission` after adjusting the table, then db2 would complain about the permission.

Reply:
ha, this is really interesting finding! I don't know this before, I thought we don't use column access control, so should not have to ...